Dockerfile

```diff
@@ -1,33 +1,14 @@
-FROM python:3.7-stretch as requirements
+FROM neuromation/minimal-base:0.9.7
 
-RUN pip install --user \
-    awscli google-cloud-storage crcmod
+RUN cd /usr/local/bin && \
+     wget https://storage.googleapis.com/kubernetes-release/release/v1.8.0/bin/linux/amd64/kubectl && \
+     chmod +x ./kubectl && \
+     kubectl version --client
+RUN apk add --no-cache rclone
```

Comment:
we have rclone in minimal-base, don't we?

Reply:
Ok, I can merge them into one